etna/transforms/math/differencing.py

```diff
@@ -81,10 +80,10 @@ def _get_column_name(self) -> str:
 
     def get_regressors_info(self) -> List[str]:
         """Return the list with regressors created by the transform."""
+        if self.in_column_regressor is None:
+            raise ValueError("Fit the transform to get the correct regressors info!")
         if self.inplace:
```

Comment:
Do we really need to check `inplace` after `self.in_column_regressor is None`?

Reply:
Yes we do because in case of `inplace=True` we return the empty list